cpp/msbuild/ice.test.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27703.2026
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.31321.278
```

Comment:
I updating this version okay?

Reply:
I think so, VS updated it when I open the project